Corona-Warn-App/src/main/res/layout/contact_diary_add_location_fragment.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Did you run autoformatter on this? the identation seems off.

Reply:
Looks like I didn't :) I'll format it.